tools/java/BUILD

```diff
@@ -18,8 +18,8 @@ if CONFIG.OS == "linux" or CONFIG.OS == "darwin":
     java_toolchain(
         name = "toolchain",
         hashes = [
-            "81bf8bee66555b8177fb4c4dd770ee9c1d243bced4bade4661f10fb1b1cf24be",
-            "a357054504bd55cceec52a9468834da5d12448e2b7779b6a1f24367d20454a33",
+            "bcd6c66f3c8a8b78c2caf7e0006be5598c365a2245120b6e8af9096d7c495c34",
```

Comment:
why are these changing?

Reply:
I'm not sure

Reply:
I suppose because it's `downloads/latest/`, i.e. presumably they have released a patch release to corretto 11.
We should either find a pinned URL or lose these hashes (because this is gonna keep happening)

Reply:
YEah and 3 days ago they pushed: https://github.com/corretto/corretto-11/commit/d7aaa4ab00a18313663f71cf40e97c6f78eabab3

Reply:
Yeah okay they suggest using versioned releases from their github. Will do that instead.